app/applications-tab/catalog/route.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model: function(params) {
+    return Ember.$.ajax('/catalog/templates', 'GET').then((response) => {
+      var data = _.sortBy(response.data, (i) => {
+        return i.name.toLowerCase();
+      });
+      var responseData  = Ember.Object.create({
+          catagories: this.buildCategories(response.data)
+        });
+      if (params.type === 'all') {
```

Comment:
'all' should probably be a `const` defined somewhere and a more special word that's unlikely to accidentally be put in as a category...